user_docs/en/userGuide.md

```diff
@@ -3232,6 +3232,22 @@ You will be asked to confirm before all trusted fingerprints are deleted.
 
 This option is only available if there are trusted fingerprints stored in your configuration.
 
+#### Local Image Description Settings Panel {#LocalCaptionerSettings}
+
+This panel provides options to customize the behavior and default settings for the local image description feature.
+You can configure how and when the model used for describing images is loaded.
+
+##### Model Path {#LocalCaptionerSettingsModelPath}
+
+Use this edit box to specify the path from which the model should be loaded.
+This allows you to use a custom model located on your local system.
```

Comment:
I don't think we want a user visible setting like this.
For now, developers can change their settings config or write an add-on.
There should be no need for end users to change the path of their model directly.
Let's just ship with one model, and a proper user friendly model manager later